examples/chapter1/CMakeLists.txt

```diff
@@ -23,66 +23,117 @@ PROJECT(${PROJ_NAME})
 
 # Include directories with headers
 #---------------------------------------------------#
-INCLUDE_DIRECTORIES( ${CMAKE_CURRENT_SOURCE_DIR}/include )
-INCLUDE_DIRECTORIES( ${KLEPSYDRA_INCLUDE_DIRS} )
-
 ADD_EXECUTABLE(${PROJ_NAME}_1 ${${PROJ_NAME}_HEADERS} src/example1.cpp )
 
 TARGET_LINK_LIBRARIES(${PROJ_NAME}_1
+  PUBLIC
   ${KLEPSYDRA_CORE_LIBRARIES}
-  ${CMAKE_THREAD_LIBS_INIT}
   atomic
```

Comment:
Try deleting the this line too. It should compile.

Reply:
I tried it but it didn't compile

Reply:
After the kpsr-core update in my PC this works . So, done